Gordon360/Services/NewsService.cs

```diff
@@ -25,26 +35,41 @@ public NewsService(IUnitOfWork unitOfWork)
         /// controller
         /// </summary>
         /// <param name="newsID">The SNID (id of news item)</param>
+        /// <param name="needsImageData">Whether or not the place from which
+        /// Get was invoked needed to get the image path or image base64 data. For
+        /// calls from editPosting, path is needed. For calls from uploadImage,
+        /// path is needed. But for calls from the controller (frontend), the data is needed.</param>
         /// <returns>The news item</returns>
-        public StudentNews Get(int newsID)
+        public StudentNews Get(int newsID, bool needsImageData)
```

Comment:
You don't currently use `needsImageData` at all it looks like - you just always read and return the image if the path exists, even if `needsImageData` is false.

Reply:
Correct- in the file itself, no other method called for it. The only place is a request from the controller.